### PR TITLE
Migrated TFS 2010 Bug Reporting Guide

### PR DESCRIPTION
**Pull request by gocGrabinski**
Added bug reporting guide for TFS 2010

### CODE REVIEW

File: source-control/Bug_Reporting_TFS_2010.md
Comment (s-laugh) [inline source-control/Bug_Reporting_TFS_2010.md]: Not all these links work. I don't think we need all of them either; probably just the first level.

File: source-control/Bug_Reporting_TFS_2010.md
Comment (s-laugh) [inline source-control/Bug_Reporting_TFS_2010.md]: The images aren't loading. You'll want to upload them to the `assets` folder.

File: source-control/Bug_Reporting_TFS_2010.md
Comment (s-laugh) [inline source-control/Bug_Reporting_TFS_2010.md]: Image not loading. Should upload it to `assets` folder.

File: source-control/Bug_Reporting_TFS_2010.md
Comment (s-laugh) [inline source-control/Bug_Reporting_TFS_2010.md]: This pointed to a dead link, probably just remove it.
```suggestion
```

File: source-control/Bug_Reporting_TFS_2010.md
Comment (s-laugh) [inline source-control/Bug_Reporting_TFS_2010.md]: This pointed to a dead link, probably just remove it.
```suggestion
```

File: source-control/Bug_Reporting_TFS_2010.md
Comment (s-laugh) [inline source-control/Bug_Reporting_TFS_2010.md]: Another previously dead link.
```suggestion
  * Optionally they can be populated manually via the Global List.
```

File: source-control/Bug_Reporting_TFS_2010.md
Comment (s-laugh) [inline source-control/Bug_Reporting_TFS_2010.md]: Images not loading. Should upload it to `assets` folder.

File: source-control/Bug_Reporting_TFS_2010.md
Comment (s-laugh) [inline source-control/Bug_Reporting_TFS_2010.md]: Point to updated/migrated doc
```suggestion
Visit our [TFS 2015 Use and Configuration Recommendation](https://esdc-devcop.github.io/recommendations/tools/tfs2015.html) for more details on Team Queries and naming convention.
```

File: source-control/Bug_Reporting_TFS_2010.md
Comment (s-laugh) [inline source-control/Bug_Reporting_TFS_2010.md]: Image not loading. Should upload it to assets folder.

File: source-control/Bug_Reporting_TFS_2010.md
Comment (s-laugh) [inline source-control/Bug_Reporting_TFS_2010.md]: Image not loading. Should upload it to assets folder.

**Review by s-laugh — CHANGES_REQUESTED** [review]
Looks good, mostly just broken links and images.

**Comment by gocGrabinski** [comment]
All done just waiting for final review to complete PR.

File: source-control/Bug_Reporting_TFS_2010.md
Comment (s-laugh) [inline source-control/Bug_Reporting_TFS_2010.md:472]: Not Blocking: These are being bundled into a single paragraph, I'm not sure if that was intend or not.

**Review by s-laugh — CHANGES_REQUESTED** [review]
Images look good now!
Still some broken links from the previous review, and one additional thing I noticed

File: source-control/Bug_Reporting_TFS_2010.md
Comment (gocGrabinski) [reply source-control/Bug_Reporting_TFS_2010.md:472]: As intended.

**Comment by gocGrabinski** [comment]
Should be good to go now.

File: source-control/Bug_Reporting_TFS_2010.md
Comment (s-laugh) [reply source-control/Bug_Reporting_TFS_2010.md]: This dead link is still present.

File: source-control/Bug_Reporting_TFS_2010.md
Comment (s-laugh) [reply source-control/Bug_Reporting_TFS_2010.md]: This link still isn't updated.

**Comment by s-laugh** [comment]
Just 2 dead links got missed.

**Comment by gocGrabinski** [comment]
Done ( I hope ) :-)

**Comment by s-laugh** [comment]
Sorry, I made that a little confusing, this one actually has an updated link that should be used https://esdc-devcop.github.io/recommendations/tools/tfs2015.html

https://github.com/esdc-devcop/guides/pull/13#discussion_r349702608

**Comment by gocGrabinski** [comment]
Done ( I hope ) :-) part II

**Review by s-laugh — DISMISSED** [review]

**Comment by s-laugh** [comment]
Looks like your first two commits didn't get signed, which is blocking the merge... :(
Hopefully you figured out your local Git issues, because you can run this to "re-sign" them https://superuser.com/a/1123928

**Comment by s-laugh** [comment]
Scratch that... you did sign them, you just haven't uploaded you public key to GitHub.
https://help.github.com/en/enterprise/2.15/user/articles/adding-a-new-ssh-key-to-your-github-account

**Comment by gocGrabinski** [comment]
Added my public key - but you sure my commit was signed before ? I used git hub web editor and commit. Not sure those would sign the commit and if it did what key did it sign it with ?

**Comment by gocGrabinski** [comment]
@s-laugh - got my local git back into proper (?) working order. Only change is that I moved the images into sub directory. Should be able to merge as I signed the commit locally ... let's see how this goes.

**Review by s-laugh — APPROVED** [review]

**Pull request merged**
